experimental/piranha_playground/rule_inference/template_parser.py

```diff
@@ -19,6 +19,12 @@ class TemplateParser:
         "java": "java",
         "kt": "kotlin",
     }  # This is necessary because get_parser and piranha expect different naming conventions
+    comments_node_names = {
+        "java": "line_comment",
+        "go": "comment",
+        "kt": "comment",
```

Comment:
I think its line_comment in later kotlin version. I think if u bump up to latest release this might not work.